networkx/algorithms/covering.py

```diff
@@ -0,0 +1,119 @@
+#    Copyright 2016 NetworkX developers.
+#    Copyright (C) 2004-2016 by
+#    Nishant Nikhil <nishantiam@gmail.com>
```

Comment:
You didn't publish this work in 2004, so this copyright line definitely isn't right.